lib/Doctrine/Common/Lexer/AbstractLexer.php

```diff
@@ -246,7 +246,7 @@ protected function scan($input)
     {
         static $regex;
 
-        if ( ! isset($regex)) {
+        if (!isset($regex)) {
```

Comment:
This change is invalid - we follow https://github.com/doctrine/coding-standard which requires this space.